src/console/CmdProcessor.cpp

```diff
@@ -109,7 +110,9 @@ void CmdProcessor::calColumnWidths(
                     break;
                 }
                 case cpp2::ColumnValue::Type::str: {
-                    size_t len = col.get_str().size();
+                    size_t wlen = utf8Conv.from_bytes(col.get_str()).size();
+                    size_t diff = (col.get_str().size() - wlen) / 2;
+                    size_t len = col.get_str().size() - diff;
```

Comment:
utf8 to gbk ?

Reply:
Later, I studied it carefully. 
The logic here should be the conversion from the byte number of the utf8 string to the byte number of the gbk string.
why utf8 to gbk ?